pkg/plugins/golang/v3/init.go

```diff
@@ -171,12 +172,25 @@ func checkDir() error {
 			if strings.HasPrefix(info.Name(), ".") {
 				return nil
 			}
+			// Allow files ending with '.md' extension
+			if strings.HasSuffix(info.Name(), ".md") && !info.IsDir() {
+				return nil
+			}
+			// Allow capitalized files except PROJECT
+			isCapitalized := true
+			for _, l := range info.Name() {
+				if !unicode.IsUpper(l) {
+					isCapitalized = false
+					break
+				}
+			}
+			if isCapitalized && info.Name() != "PROJECT" {
```

Comment:
Will not that allow init a project when the dir has the PROJECT file already? 
Then, it means that the tool cannot ensure that the source of true is consistent.

Reply:
I think this case would break earlier, mind to check?

Reply:
@camilamacedo86 According to this comment on the one you put into the issue, should I only remain the `.md` files whitelist into this PR? And add some useful capitalized files into `allowedFiles` array?

You right, I haven't tested this case.
It break earlier here: https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/cli/cmd_helpers.go#L225

Reply:
Yeah I was pretty sure it did.